airlock/templates/file_browser/file.html

```diff
@@ -43,45 +43,91 @@
           </form>
         {% endif %}
       {% elif is_output_checker %}
+        <div>
+          {% if not file_reset_review_url %}
+            <span class="file-status file-status--warning">No status</span>
+          {% elif file_approve_url %}
+            <span class="file-status file-status--danger">Rejected</span>
```

Comment:
The button says "Request changes" button, can this say "changes requested" (or something similar) rather than "rejected". (There's a ticket to do the more complicated work to change the state from rejected to changes-requested in the backend)

Reply:
Good spot, this bit was done before I was told that was changing 👍